testsetup/testsetup.go

```diff
@@ -25,7 +25,8 @@ const (
 
 // user environment variables
 const (
-	EnvLogfileTmpl = "GOVIM_LOGFILE_TMPL"
+	EnvLogfileTmpl           = "GOVIM_LOGFILE_TMPL"
+	EnvDisableHighlightDiags = "GOVIM_DISABLE_HIGHLIGHTDIAGS"
```

Comment:
I don't think we need the environment variable master override in this case.

Instead we should just add a regular `Config` value to disable this:

https://pkg.go.dev/github.com/govim/govim/cmd/govim/config?tab=doc#Config

Reply:
fixed